packages/dev/core/src/XR/features/WebXRImageTracking.ts

```diff
@@ -132,14 +130,6 @@ export class WebXRImageTracking extends WebXRAbstractFeature {
         return super.detach();
     }
 
-    /**
-     * Check if the needed objects are defined.
-     * This does not mean that the feature is enabled, but that the objects needed are well defined.
-     */
-    public isCompatible(): boolean {

```

Comment:
Was this something you previously added that is no longer needed or something?

Reply:
This was added in the original implementation from Raanan, however this is just an interface which doesn't actually need to be implemented as a hardened JS class.

We also already check for whether getImageTrackingResults and getTrackedImageScores are implemented by the WebXR side before attempting to use the feature, so this is kind of just an extra safety check that isn't actually explicitly needed.

For these reasons I decided to remove it.  If we want to keep this here to prevent attempting to initialize the feature then I'd need to just define a dummy class somewhere for the native side since we don't actually use this anywhere natively.

Reply:
For other WebXR features we don't explicitly check the classes when attaching like we are doing in this case.

Reply:
This was added when it was in beta mode and was not available in most ar-supporting browsers. This has changed since then. There is also the extra check of "required features".

This is just an initial test to see if it is even viable to enable this feature. Is there a different object you can check instead?